src/repositories.js

```diff
@@ -1,18 +1,37 @@
 /**
- * Usage as below.
- *
- * Required properties are id (number), name (string).
- *
- * ```
- * const repositoryStore = require('./repositories').instance;
- *
- * const repo = { id: 1, name: 'foo-bar' };
- *
- * repositoryStore.set(repo.id, repo);
- * ```
+ * @constant repositoryStore - A global map of repositories that are assocated with the Tako GitHub App installation.
  */
 const repositoryStore = new Map();
```

Comment:
I could well be mistaken, but I _think_ this could now be changed to a simple ol' array, unless there are some benefits we get from using a `Map` that I'm not aware of.

Reply:
Hells yeah